components/announcement_bar/no_internet_connection/no_internet_connection.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+
+import GenericModal from 'components/generic_modal';
+
+import './no_internet_connection.scss';
+import NoInternetConnectionSvg from './no-internet-connection-svg';
+
+type NoInternetConnectionProps = {
+    onHide: () => void;
+};
+const NoInternetConnection: React.FC<NoInternetConnectionProps> = (props: NoInternetConnectionProps) => {
+    return (
+        <GenericModal
+            onHide={props.onHide}
+            modalHeaderText=''
+        >
+            <div className='noInternetConnection__container'>
+                <div className='noInternetConnection__image'>
+                    <NoInternetConnectionSvg/>
+                </div>
+                <span className='noInternetConnection__noAccessToInternet'>
+                    <FormattedMessage
+                        id='announcement_bar.warn.no_internet_connection'
+                        defaultMessage='Looks like you do not have access to the internet.'
+                    />
+                </span>
+                <span className='noInternetConnection__contactSupport'>
+                    <FormattedMessage
+                        id='announcement_bar.warn.contact_support'
+                        defaultMessage='In order to renew, you need to contact support.'
```

Comment:
Just confirming, this isn't the bar that would pop on an on-prem installation correct? If so, this message might be confusing to them if their connection isn't working.

Reply:
Agreed, brought this up in the channel [here](https://community-daily.mattermost.com/private-core/pl/sujnod5c7fyg7fu9tpwsaxd75o) and a new copy was suggested: 

```
It looks like you do not have access to the internet.

To renew your license, contact support at support@mattermost.com.

<Contact support>
```